Sources/Datadog/DatadogInternal/DatadogCoreProtocol.swift

```diff
@@ -11,6 +11,53 @@ public internal(set) var defaultDatadogCore: DatadogCoreProtocol = NOPDatadogCor
 /// A Datadog Core holds a set of features and is responsible of managing their storage
 /// and upload mechanism. It also provides a thread-safe scope for writing events.
 public protocol DatadogCoreProtocol {
+    /// Registers a Feature instance.
+    ///
+    /// A Feature collect and transfer data to a Datadog Product (e.g. Logs, RUM, ...). A registered Feature can
```

Comment:
IMO, if we add doc string for method signatures in `protocol` definition, there is no need to repeat it in `extension` when adding the actual conformance - the IDE should infer it. However, it is fine with me if we decide to do so - just let's ensure that both are in sync.

Typo:
```suggestion
    /// A Feature collects and transfers data to a Datadog Product (e.g. Logs, RUM, ...). A registered Feature can
```